### ISSUE TITLE
Don't write to *.innerHTML

### ISSUE THREAD

**Issue by Maingron** [Maingron/MainOS#21]
Don't use *.innerHTML = `OR even` += "something" . Instead spawn new HTML elements like described here: https://github.com/Maingron/MainOS/security/code-scanning/1997?query=ref%3Arefs%2Fheads%2Fmaster

This will improve scores, flexibility and accessibility. Might improve some other things too 🤷‍♂️

>Assignments to [innerHTML](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML)/[outerHTML](https://developer.mozilla.org/en-US/docs/Web/API/Element/outerHTML) properties or calls to [insertAdjacentHTML](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML) method manipulate DOM directly without any sanitization and should be avoided. Use document.createElement() or similar methods instead.

**Comment by Maingron** [comment]
Does anyone have time to replace all the innerHTMLs with appendChild() or something similar?  
Currently there's a list of innerHTML writes but I'd suggest using CTRL F https://github.com/Maingron/MainOS/security/code-scanning?query=is%3Aopen+innerHTML
